src/shashmap.rs

```diff
@@ -29,10 +28,10 @@ impl Shashmap {
     /// * `tx` - `Transaction` where inputs are inserted, and outputs are removed
     pub fn unspend_transaction(&mut self, _tx: &Transaction) {}
 
-    /// Return the `Block` id based on `Slip`
+    /// Return the `Block` id based on `OutputSlip`
     ///
-    /// * `slip` - `&Slip` as key
-    pub fn slip_block_id(&self, slip: &Slip) -> Option<&i64> {
+    /// * `slip` - `&OutputSlip` as key
+    pub fn slip_block_id(&self, slip: &SlipID) -> Option<&OutputSlip> {
```

Comment:
This function name doesn't make any sense with the proposed changes. Should be something like `get_output_slip`